examples/04-RNN-examples.ipynb

```diff
@@ -4450,7 +4450,7 @@
     },
```

Comment:
<p>It looks like there were no code changes in this notebook. Can you try to revert the changes to this file? We usually try to keep the git history/diffs as minimal as possible :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/2008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>